client/test/e2e/services/services.spec.js

```diff
@@ -0,0 +1,106 @@
+/*global describe, it, element, by, beforeEach, inject, browser */
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+var expect = chai.expect;
+
+var FormUtils = require('../shared/FormUtils');
+
+describe('The Projects Module', function () {
+  // shared methods
+  var path = '#/services';
+  var SERVICE = {
+    name : 'Zebra Service',
+  };
+
+  //To obtain the rank of a random element to the Service list
+  function aleatoire(N) { 
```

Comment:
In English, "aleatoire" is "random".  Since you have used this in several e2e tests, I created a helper function for us to use in `helpers.js`.  [Here is the source code, if you are curious](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/test/e2e/shared/helpers.js#L11).  If you import helpers like the comment above you can do this:
```js

// this is the same as aleatoire(DEFAULT_SERVICE).
var SERVICE_RANK = helpers.random(DEFAULT_SERVICE);
```